sample-apps/discounts-tutorial/web/frontend/pages/Volume/[id].jsx

```diff
@@ -0,0 +1,305 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useForm, useField } from "@shopify/react-form";
+import { Redirect } from "@shopify/app-bridge/actions";
+import { useAppBridge } from "@shopify/app-bridge-react";
+import {
+  ActiveDatesCard,
+  CombinationCard,
+  DiscountClass,
+  DiscountMethod,
+  MethodCard,
+  UsageLimitsCard,
+  onBreadcrumbAction,
+} from "@shopify/discount-app-components";
+import {
+  Banner,
+  Card,
+  Layout,
+  Modal,
+  Page,
+  PageActions,
+  Stack,
+  Spinner,
+  TextField,
+  TextContainer,
+} from "@shopify/polaris";
+
+import {
+  useUpdateDiscount,
+  useDeleteDiscount,
+  useDiscount,
+} from "../../hooks/";
+
+const METAFIELD_NAMESPACE = "discounts-tutorial";
+const METAFIELD_CONFIGURATION_KEY = "volume-config";
+
+export default function VolumeNew() {
+  const app = useAppBridge();
+  const redirect = Redirect.create(app);
+  const { id } = useParams();
+  const { discount, isLoading } = useDiscount(id);
+  const [updateDiscount, { isLoading: updateInProgress }] = useUpdateDiscount(
+    discount?.method
+  );
+  const [deleteDiscount, { isLoading: deleteInProgress }] = useDeleteDiscount(
+    discount?.method
+  );
+  const [deleteModalActive, setDeleteModalActive] = useState(false);
+  const [successActive, setSuccessActive] = useState(false);
+  const [errors, setErrors] = useState([]);
+
+  const {
+    fields: {
+      discountTitle,
+      discountCode,
+      discountMethod,
+      combinesWith,
+      usageLimit,
+      appliesOncePerCustomer,
+      startsAt,
+      endsAt,
+      configuration,
+    },
+    submit,
+    submitting,
+    dirty,
+  } = useForm({
+    fields: {
```

Comment:
Are you missing [some fields](https://github.dev/Shopify/function-examples/blob/e7d69934d0dae0ad94f89342b17416137cadd70a/sample-apps/discounts-tutorial/web/frontend/pages/Volume/new.jsx#L72-L76) here?